de_identify_csv.py

```diff
@@ -0,0 +1,80 @@
+import argparse
```

Comment:
I'm less fond of us copying these tools that don't get run as part of the pipeline from project to project. Could we consider putting them in infrastructure or a tools library to get a bit more standardisation and visibility of the tools that are available?

Reply:
I think PipelineInfrastructure is the obvious place to put them, and this script is in a position where we can do that now (this is the third project we've used it on, with the only modification being the conversion to Firestore uuid tables).

Reply:
Will remove in a future PR once a relevant PR in Pipeline-Infrastructure has been accepted